Fastfile

```diff
@@ -1,175 +1,192 @@
 fastlane_version "2.30.1"
-
 default_platform :ios
 
-  before_all do
-    if ENV['SLACK_URL'] == nil
-      ENV['SLACK_URL'] = ENV['TAB_SLACK_WEBHOOK_URL']
-    end
-  end
-
-  lane :test do
-    _setup()
-    skip_slack = ENV['SCAN_SLACK_CHANNEL'].to_s.strip.empty?
-    scan(skip_slack: skip_slack)
-  end
-
-  lane :ui_test do
-    _setup()
-    skip_slack = ENV['SCAN_SLACK_CHANNEL'].to_s.strip.empty?
-    scan(skip_slack: skip_slack,
-         devices: ENV['TAB_UI_TEST_DEVICES'],
-         output_types: ENV['TAB_REPORT_FORMATS'],
-         scheme: ENV['TAB_UI_TEST_SCHEME'])
-  end
-
-  lane :deploy_to_hockey do
+# --------- Before any lane runs --------- #
+
+before_all do
+  if ENV['SLACK_URL'] == nil
+    ENV['SLACK_URL'] = ENV['TAB_SLACK_WEBHOOK_URL']
+  end
```

Comment:
Pretty certain this can just be
```ruby
ENV['SLACK_URL'] ||= ENV['TAB_SLACK_WEBHOOK_URL']
```
`||=` is my fave Ruby operator. Only sets the thing on the left to the value on the right if the thing on the left is nil. Super useful for doing lazy initialisation.

Reply:
Ooh fancy!

Reply:
Updated